src/coreclr/pal/inc/rt/ntimage.h

```diff
@@ -230,7 +230,6 @@ typedef struct _IMAGE_FILE_HEADER {
 #define IMAGE_FILE_MACHINE_AM33              0x01d3
 #define IMAGE_FILE_MACHINE_POWERPC           0x01F0  // IBM PowerPC Little-Endian
 #define IMAGE_FILE_MACHINE_POWERPCFP         0x01f1
-#define IMAGE_FILE_MACHINE_IA64              0x0200  // Intel 64
```

Comment:
This should be kept

Reply:
Yeah if the goal is to represent possible values I agree, and then anything that handles its appearance should stay

Reply:
There is a lot of unused definitions in this file. It was copied from Windows.